samples/bluetooth/ble_pwr_profiling/src/main.c

```diff
@@ -197,12 +197,12 @@ static void notification_send(void)
 			.p_data = char_value,
 		};
 
-		err = sd_ble_gatts_hvx(conn_handle, &hvx_params);
-		if ((err != NRF_SUCCESS) &&
-		    (err != NRF_ERROR_INVALID_STATE) &&
-		    (err != NRF_ERROR_RESOURCES) &&
-		    (err != BLE_ERROR_GATTS_SYS_ATTR_MISSING)) {
-			LOG_ERR("sd_ble_gatts_hvx failed, nrf_error %#x", err);
+		nrf_err = sd_ble_gatts_hvx(conn_handle, &hvx_params);
+		if (nrf_err &&
```

Comment:
Let's keep it this way for readability.
```suggestion
		if (nrf_err != NRF_SUCCESS &&
```